web_programming/currency_converter.py

```diff
@@ -176,9 +177,9 @@ def convert_currency(
     from_: str = "USD", to: str = "INR", amount: float = 1.0, api_key: str = API_KEY
 ) -> str:
     """https://www.amdoren.com/currency-api/"""
-    params = locals()
-    params["from"] = params.pop("from_")
+    params = dict(from=from_, to=to, amount=amount, api_key=api_key)
```

Comment:
This raises a syntax error because `from` is a reserved keyword in Python (?) so its better to use the regular dictionary syntax here.
```suggestion
    params = {
        "from": from_,
        "to": to,
        "amount": amount
        "api_key": api_key
    }
```

Reply:
Oops, thanks. No problem.